blazor/message/customization.md

```diff
@@ -106,7 +106,7 @@ The below DOM structure is required to display the simple message with content.
 
 ```
 
-The below DOM structure is required to display the simple message with content and severity icon.
+The following DOM structure is required to display the simple message with the content and severity icon.
```

Comment:
template.md file does not having below changes
In the **following** sample instead of In the **below** sample

Reply:
Replaced the content